QPM/Providers/RemoteQPMDependencyResolver.cs

```diff
@@ -325,26 +326,44 @@ public void ResolveDependency(in Config myConfig, in RestoredDependencyPair pair
             WebClient client = new();
             // soName is dictated by the overriden name, if it exists. Otherwise, it is this.
             var soName = sharedConfig.Config.Info.GetSoName(out var overrodeName);
+            var linker = new SymLinker.Linker.Linker();
```

Comment:
Use a `using` statement instead of a full name here. In addition, only make a `Linker` instance if a symlink is necessary, no need to make it for every pass.

In fact, there's no clear reason why you have to make it here and pass it into a method instead of just making it in the method (since this is still only called once)

If your goal is to have one stick around for a pass through all the .so files (which would be somewhat ideal) create it on ..ctor, ex, as a private static field instead, or on construction of this particular instance and have it as an instance field.